### DISCUSSION TITLE
Load individual resources as JSON?

### DISCUSSION THREAD

**Discussion by nephix** [i18next/next-i18next#1856] [Q&A] [answered]
Hey guys,

is it possible to load individual resouces as JSON from a specific location, like supported [here](https://www.i18next.com/overview/configuration-options#languages-namespaces-resources) from `i18next`?

I tried something like:

```
resources: {
  en: {
    email: emailEN,
  },
  de: {
    email: emailDE,
  },
},
```

in the `next-18next.config.js` but the resources specified there could not be found. 

My use case is that I don't have a single locale path, I have a monorepo with a locale package that provides json files. Every application can use these common locale files as well as specific locale files locally in the application

**Comment by isaachinman** [comment, marked as answer]
Yes, see #1642.

**Reply by nephix** [reply]
How does this work under the hood, is it merged with the default path at `public/locales`?
